ledger-core/virtual/integration/method_test.go

```diff
@@ -532,3 +532,191 @@ func TestVirtual_CallContractFromContract_Unordered(t *testing.T) {
 	server.WaitActiveThenIdleConveyor()
 	mc.Finish()
 }
+
+// A.New calls ordered B.Foo
+func TestVirtual_CallMethodFromConstructor_Ordered(t *testing.T) {
+	t.Log("C5091")
+
+	mc := minimock.NewController(t)
+
+	server, ctx := utils.NewUninitializedServer(nil, t)
+	defer server.Stop()
+
+	runnerMock := logicless.NewServiceMock(ctx, mc, func(execution execution.Context) string {
+		return execution.Request.CallSiteMethod
+	})
+	server.ReplaceRunner(runnerMock)
+	server.Init(ctx)
+	server.IncrementPulseAndWaitIdle(ctx)
```

Comment:
no need to increment pulse here

Reply:
it panics without increment 
<img width="1616" alt="Screenshot 2020-06-17 at 16 11 40" src="https://user-images.githubusercontent.com/6708245/84902382-6f3c1880-b0b5-11ea-88c6-7e3fa76ebf34.png">